src/PostgREST/App.hs

```diff
@@ -117,9 +119,22 @@ transactionMode proc action =
          else HT.Write
     _ -> HT.Write
 
+getRawMediaTypesForApiRequest :: AppConfig -> ApiRequest -> [ContentType]
+getRawMediaTypesForApiRequest conf request
+  | configRawMediaEndpoints conf == M.empty
+    = decodeContentType <$> configRawMediaTypes conf
+  | otherwise
+    = let targetText :: Text
+          targetText =  targetIdentifierText $ iTarget request
+          requestContentTypes = decodeContentType
+            <$> fromMaybe [] (M.lookup targetText $ configRawMediaEndpoints conf)
+          defaultContentTypes = (decodeContentType
+            <$> fromMaybe [] (M.lookup "any" $ configRawMediaEndpoints conf))
```

Comment:
@steve-chavez I was wondering if we should take an "any" approach, where certain mime types are added to all requests, or an "otherwise" approach instead, which would apply the mime types only if no other setting matches the target. What do you think is best?

Reply:
Oh, sorry I'm a bit lost here. Could you provide a sample config with the `any`?

I'm thinking you mean something like:

```
raw-media-endpoints {
  any = "image/png"
}
```

But not sure. A config with the `otherwise` would also help me understand the idea.

Reply:
The intent is to give the possibility to define a catch-all clause in the config so that a default mime type can be made a raw media type for all endpoints.

What I am trying to assess is whether you think this functionality is needed, and also whether such defaults should work as `AND` or `OR`.

With `any`, endpoints would get both the `any` types, and the types for their specific endpoint: 
`raw-media-endpoints {
  any = "mime/foo"
  images = "mime/bar"
}`
a request to `/images` would accept both `mime/foo` and `mime/bar` as raw media types
a request to any endpoint not specified in raw-media-endpoints would only accept `mime/bar` 

instead, what I mean with `otherwise` is, given this config
`raw-media-endpoints {
  images = "mime/foo"
  otherwise = "mime/bar"
}`
a request to `/images` would accept `image/jpg` only. while a request to an endpoint not specified in `raw-media-endpoints` would accept `image/png`

I hope I did a better job of explaining what I mean.

Reply:
@Dansvidania Ah, I understand the `any/otherwise` proposal now. One problem I see is that they could make all the other responses fail in a browser, same as `raw-media-types`. Say I have an example html page like [the one I mention here](https://gitter.im/begriffs/postgrest?at=5eecf39ab8152d34845082e9), plus another one that I'd have in `/rpc/admin.html`. I could do this with the config:

```
raw-media-endpoints {
  any = "text/html"
}
```

Calling the html endpoints would work but all the others would result in the same issue as https://github.com/PostgREST/postgrest/issues/1462 (`text/html requested but more than one column was selected`).

It'd be better to ask the user to be specific and instead do:

```
raw-media-endpoints {
  /rpc/index.html = "text/html"
  /rpc/admin.html = "text/html"
}
```

In general, I think these endpoints(html or image) will be few. So I'd go for no `any/otherwise` for now. If there's a need for a more generic config we could add that later. What do you think?